packages/client/src/utils/index.ts

```diff
@@ -95,6 +95,24 @@ export const isValidPostalcode = (value?: string) => {
   return !!(value && value.toString().trim().match(postalCodeRegex));
 };
 
+/**
+ *
+ * This function transforms some cases where a user for example makes a typo or makes a strange space combination to a valid option.<
```

Comment:
Can you add examples at "some cases where a user for example makes a typo" and "makes a strange space combination to a valid option"